ppr-ui/src/components/common/RegistrationsWrapper.vue

```diff
@@ -863,109 +863,114 @@ export default defineComponent({
     const handleRegTableNewItem = async (val: RegTableNewItemI) => {
```

Comment:
Hide white space changes for easier reviewing 😁 

![Screen Shot 2023-04-18 at 11 41 21 AM](https://user-images.githubusercontent.com/53542131/232872825-f961ea20-eea4-4e30-9d5f-8c451aa4804d.png)